orchestra/communication/staffing.py

```diff
@@ -39,8 +41,6 @@ def handle_staffing_response(worker, staffing_request_inquiry_id,
         response (orchestra.models.StaffingResponse):
             StaffingResponse object that has been created for the worker
     """
-
-    # TODO(kkamalov): add proper docstring
```

Comment:
the world's longest gap from TODO to DIDN'TDO

🌹

Reply:
:D